platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -392,7 +401,7 @@ class HangingProtocolService {
     }
 
     this.customImageLoadPerformed = false;
-    const { type: layoutType } = stageModel.viewportStructure;
+    const { layoutType } = stageModel.viewportStructure;
```

Comment:
Is it necessary to rename the field name?  Probably it is, given the use of type as a keyword.

Reply:
Yes it was actually a bug, it is defined as layoutType in the protocols